model/src/main/java/org/mskcc/cmo/metadb/model/igo/IgoLibrary.java

```diff
@@ -0,0 +1,110 @@
+package org.mskcc.cmo.metadb.model.igo;
```

Comment:
If `../Library` is removed then this will be renamed from `IgoLibrary` to `Library`